docker/blockchain/Dockerfile

```diff
@@ -27,17 +26,25 @@ RUN make VERSION=${VERSION} WITH_CLEVELDB=true install
 
```

Comment:
I think this could be an issue as well. It looks like the binary will be built on an x86_64 machine, and could potentially cause issues on an arm64 machine.

Reply:
RUN is a shell executed on the container not the host.  So, when github actions build a arm64 that container will be building for the arm64 arch. In this case, the `make` is being executed on the container.

I removed the whole ARG passing for builds, because we were passing the arch of the host machine not the container platform.